src/assets/scss/_settings.scss

```diff
@@ -164,7 +167,7 @@ $accent-color: $seamint;
 // $info-color: #a0d3e8;
 
 // We use these to control various global styles
-$body-bg: #fff;
+$body-bg: $jerseywhite;
```

Comment:
Let's leave `$body-bg` as white— if you need a specific background color, that should be set in the component that needs it. Let me know if that makes sense.